#==================================================================================================
# Upstream patch to fix FONT_COLOR_GLYPHS, identified by our own Ryan Schmidt:
#
# FONT_COLOR_GLYPHS appears to be backwards. You're using kCTFontTraitColorGlyphs on less than
# 10.8 and kCTFontColorGlyphsTrait on 10.8 and later, yet it is kCTFontColorGlyphsTrait that is
# available in 10.7 and later and deprecated in newer systems and kCTFontTraitColorGlyphs which
# is available in 10.8 and later.
#
# Issue: https://gitlab.freedesktop.org/cairo/cairo/-/issues/810
#==================================================================================================
--- src/cairo-quartz-font.c.orig
+++ src/cairo-quartz-font.c
@@ -77,10 +77,10 @@
 
 #if MAC_OS_X_VERSION_MIN_REQUIRED < 1080
 #define FONT_ORIENTATION_HORIZONTAL kCTFontHorizontalOrientation
-#define FONT_COLOR_GLYPHS kCTFontTraitColorGlyphs
+#define FONT_COLOR_GLYPHS kCTFontColorGlyphsTrait
 #else
 #define FONT_ORIENTATION_HORIZONTAL kCTFontOrientationHorizontal
-#define FONT_COLOR_GLYPHS kCTFontColorGlyphsTrait
+#define FONT_COLOR_GLYPHS kCTFontTraitColorGlyphs
 #endif
 
 static void
